Source/WebKit/Shared/RemoteLayerTree/RemoteLayerBackingStore.h

```diff
@@ -58,7 +58,7 @@ class RemoteLayerBackingStore {
     RemoteLayerBackingStore(PlatformCALayerRemote*);
     ~RemoteLayerBackingStore();
 
-    enum class Type : uint8_t {
+    enum class Type : bool {
```

Comment:
Why is `bool` better?

Reply:
bool only has 2 valid states, so the decoder doesn't need any more metadata if an enum class uses bool as an underlying type.